lib_device_control/host/device_access_usb.c

```diff
@@ -213,14 +214,14 @@ control_ret_t control_init_usb(int vendor_id, int product_id, int interface_num)
 
   int r = usb_set_configuration(devh, 1);
   if (r < 0) {
-    fprintf(stderr, "Error setting config 1\n");
+    PRINT_ERROR("Error setting config 1\n");
```

Comment:
As "error" would otherwise be repeated twice, I would substitute "failed to set config" for "Error setting config".